### DISCUSSION TITLE
People can't list my repo

### DISCUSSION THREAD

**Discussion by abidinburak** [community/community#121646] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I have a repo about problem solving with kotlin on codeforces but no one can see my repo.
It is public and I share it but no one can search it in github.

https://github.com/abidinburak/Codeforces-Soulitons-Kotlin

I didn't understand what my mistake was. Can you help me about that?

**Comment by nikeedev** [comment]
I don't know if thats the case, but maybe the people who search that name they write Soultions, while you made a typo and wrote Soulitons.

See, when i search it with the typo (https://github.com/search?q=codeforces+soulitons&type=repositories) your repository does shows up, while if you search without the typo like probably the others who couldn't find the repository: https://github.com/search?q=codeforces+soultions&type=repositories, it wouldn't show up.

**Comment by gurjeetsinghvirdee** [comment]
Hey @abidinburak 
Just change your repo name `Codeforces-Soulitons-Kotlin` to `Codeforces-Solutions-Kotlin` it will work fine, you just made a typo mistake in naming that's why no one could able to find your repo

**Reply by abidinburak** [reply]
Thank you so much, I try everything but i did not see that mistake.

**Reply by nikeedev** [reply]
That's exactly what i first mentioned...well...

**Reply by gurjeetsinghvirdee** [reply]
Not a problem at all buddy @nikeedev 
maybe he didn't understand by your comment 
I just highlighted the text
bro @abidinburak mark his comment as answered

Happy Coding!
